keps/prod-readiness/sig-network/3070.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3070
+alpha:
+  approver: "@johnbelamaric"
```

Comment:
It seems most of people are adding John...

Feel free to change to me (also might be easier due to less time-zone difference).